### PR TITLE
Internal package.json license: GPLv3

### PR DESCRIPTION
**Pull request by dcposch**
Changed from 'Propietary' to 'GPLv3' to match `LICENSE.md`

### CODE REVIEW

**Comment by dcposch** [comment]
Alternatively we could just remove the `license` fields from each `package.json`. N1 says it's GPL in documentation & LICENSE.md, don't know if we have to specify again for each individual internal package. Thoughts?

**Comment by bengotow** [comment]
Hey @dcposch — thanks for pointing this out, we definitely need to fix this. Can you do me a favor and sign the Nylas CLA (https://nylas.com/cla)? That'll make the PR bot happy and we can land this.

It's not really necessary to have the license in there since the entire project is GPLV3, but I think these packages may be copy-pasted a lot as people create their own, and we'd like for all of those to list a license, so I vote we keep the field.

**Comment by dcposch** [comment]
@bengotow done. Should I also change the example packages, eg https://github.com/nylas/N1/blob/master/examples/N1-Filters/package.json ?

File: internal_packages/account-sidebar/package.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
   "version": "0.1.0",
   "main": "./lib/main",
   "description": "Sidebar view that shows your account and important tags",
-  "license": "Proprietary",
+  "license": "GPLv3",
```
Comment (slang800) [inline internal_packages/account-sidebar/package.json]: the [SPDX license code](https://spdx.org/licenses/) is `GPL-3.0`... npm won't recognize `GPLv3` properly

Comment (slang800) [inline internal_packages/account-sidebar/package.json]: update: fixed that in #491

**Pull request merged**